ansible/roles/operator-pipeline/templates/openshift/tasks/create-support-link-for-pr.yml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-support-link-for-pr
+spec:
+  params:
+    - name: env
+      description: Enviroment. One of [dev, qa, stage, production]
+    - name: type
+      description: Connect case type.
+      default: "CERT"
+    - name: source
+      description: Connect case source.
+      default: "tekton"
+    - name: cert_project_id
+      description: Pyxis certification project id.
+    - name: cert_project_type
+      description: Connect cert project type. Should be Operator Bundle Image for operator pipelines.
+      default: "Operator Bundle Image"
+    - name: pull_request_url
+      default: ""
+  results:
+    - name: comment
+  steps:
+    - name: create-support-link-for-pr
+      image: quay.io/redhat-isv/operator-pipelines-images
+      env:
+        - name: LINK_ENV
+          value: $(params.env)
+        - name: LINK_TYPE
+          value: $(params.type)
+        - name: LINK_SOURCE
+          value: $(params.source)
+        - name: CERT_PROJECT_ID
+          value: $(params.cert_project_id)
+        - name: CERT_PROJECT_TYPE
+          value: $(params.cert_project_type)
+        - name: PULL_REUQEST_URL
+          value: $(params.pull_request_url)
+      script: |
+        #! /usr/bin/env bash
+        set -e
+
+        echo "Follow this link if you wish to open a support case:"
+        echo "https://$URL_ENV.redhat.com/support/technology-partner/#/case/new?type=$LINK_TYPE&source=$LINK_SOURCE&cert_project_id=$CERT_PROJECT_ID&cert_project_type=$CERT_PROJECT_TYPE&pull_request_url=$PULL_REUQEST_URL" | tee $(results.comment.path)
```

Comment:
Sadly, link is broken- it contains the whitespaces 
https://.redhat.com/support/technology-partner/#/case/new?type=CERT&source=tekton&cert_project_id=610260e6d9a6bd83d7b6d5f7&cert_project_type=Operator Bundle Image&pull_request_url=https://github.com/redhat-openshift-ecosystem/operator-pipelines-test/pull/31

Reply:
That is why url encoding was added, @wcheang so wdyt, should we keep URL encoding?

Reply:
Yeah we should encode it then...sorry for the misdirection!

Reply:
Updated!